website/docs/addon_maya_ornatrix_artist.md

```diff
@@ -13,31 +14,14 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
 AYON supports Ephere's **Ornatrix** with the workflow of publishing and loading maya scene with Ornatrix data, Ornatrix groom presets and Ornatrix Alembic. It also supports to connect your target geometry with the loaded Ornatrix data (either from the maya scene or the groom preset.)
 
-## Maya Addon Setting before using AYON for Ornatrix Maya
-On the AYON Studio Settings or Project Settings enable the Ornatrix plug-ins for Maya:
-
-- Project settings > `Maya` > `Creators` > `Create Ornatrix Rig`
-- Project settings > `Maya` > `Creators` > `Create Ornatrix Cache`
-- Project settings > `Maya` > `Publishers` > `Validate Ornatrix Rig Contents`
-- Project settings > `Maya` > `Publishers` > `Validate Ornatrix Cache Contents`
-- Project settings > `Maya` > `Loaders` > `Ornatrix Cache Loader`
-- Project settings > `Maya` > `Loaders` > `Ornatrix Rig Loader`
-
-
-![Maya - Ornatrix Creators Setting](assets/maya/artist/ox_creators_maya_addon_setting.jpg)
-
-![Maya - Ornatrix Validator Setting](assets/maya/artist/ox_validator_maya_addon_setting.jpg)
-
-![Maya - Ornatrix Loader Setting](assets/maya/artist/ox_loaders_maya_addon_setting.png)
-
 ## Workflow
-AYON has two creators for Ornatrix publishing: ```Ornatrix Rig``` and ```Ornatrix Cache```.
+AYON has two creators for Ornatrix publishing: `Ornatrix Rig` and `Ornatrix Cache`.
 
 ## Ornatrix Rig
```

Comment:
I've a question about Ornatrix rig. 

So, this section doesn't mention anything about connecting rigs to pointcaches or animations. 
and I can't understand why would artists need to connect them. 

Could we elaborate on this workflow?

Reply:
Also, test notes in this PR https://github.com/ynput/ayon-maya/pull/1 look useful. could we use bits of them to write about the workflow and what artists should do and expect ?

Reply:
> I've a question about Ornatrix rigs.
> 
> So, this section doesn't mention anything about connecting rigs to pointcaches or animations. and I can't understand why would artists need to connect them.
> 
> Could we elaborate on this workflow?

it is similar to how the connect yeti rig works. When loading the animated characters and they are not at the original positions(e.g. they are sitting or running), the connect action can help the rigs to find the correct position of where the fur(e.g. hair) located.

Reply:
so, we link the animated character with its animated hair/fur/groom to be able to see the full animation.

Reply:
Thank you for your explanation. it helped me update the answer.
https://github.com/ynput/ayon-documentation/pull/237/commits/f218f5198a3ebd2a14154c0e1a5f6020bbbd50b0